manifests/database.pp

```diff
@@ -32,5 +32,6 @@
       environment => delete_undef_values($seed_env),
     }
     ~> Foreman::Rake['apipie:cache:index']
+    ~> Foreman::Rake['apipie_dsl:cache']
```

Comment:
This can be simplified to
```suggestion
    ~> Foreman::Rake['apipie:cache:index', 'apipie_dsl:cache']
```

This assumes that an `apipie:cache:index` doesn't need to trigger an `apipie_dsl:cache`.

(same in plugin.pp)

Reply:
Thank you, done.